lib/packages.js

```diff
@@ -188,13 +202,15 @@ function _buildImportAllModule() {
 
   // write importAll.js
   data = '';
-  if(imports.length > 0) {
-    imports.forEach(lib => {
-      data += 'import \'' + lib + '\';\n';
-    });
-  }
+  imports.forEach(lib => {
+    data += `import '${lib}';\n`;
+  });
+  data += 'import {start} from \'bedrock-angular\';\n';
+  data += 'start();\n';
```

Comment:
Do we have to add `start` to `importAll` or can this be done via some other file? Can web pack then pull in that other file instead of `importAll` -- thereby keeping concerns separate -- so people can pull in `importAll` on their own and then decide when to call start? Just concerned about ease of customization... we moved `importAll` into its own script to keep any other behavior out of it.